scripts/lib.star

```diff
@@ -443,29 +443,24 @@ def build_frontend_step(edition, is_downstream=False):
         ],
     }
 
-def generate_packages_docs(edition, lint=False):
+def generate_package_docs(edition, lint=False):
     if edition == 'enterprise':
         return None
 
     if lint:
```

Comment:
Can't you remove the `lint` condition now, since both branches are the same (AFAICT)?

Reply:
Almost the same :D They run different scripts. The first one will only build the other will build and lint.

Reply:
@mckn Yeah I noticed when checking locally, that there is a slight difference.